src/SQL/FindAssets.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\SQL;
+
+use Doctrine\DBAL\Driver\Connection;
+use Doctrine\DBAL\DriverManager;
+
+/**
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ */
+class FindAssets
+{
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    public function __construct(
+        $databaseHost,
+        $databasePort,
+        $databaseName,
+        $databaseUser,
+        $databasePassword
+    ) {
+        $connectionParams = [
+            'host' => $databaseHost,
+            'port' => $databasePort,
+            'dbname' => $databaseName,
+            'user' => $databaseUser,
+            'password' => $databasePassword,
+            'driver' => 'pdo_mysql',
+        ];
+        $this->connection = DriverManager::getConnection($connectionParams);
+    }
+
+    public function fetch(): \Iterator
+    {
+        $this->connection->getWrappedConnection()->setAttribute(\PDO::MYSQL_ATTR_USE_BUFFERED_QUERY, false);
+        $fetchAssetsQuery = <<<SQL
+SELECT 'code', 'categories', 'description', 'end_of_use', 'localized', 'tags'
+UNION ALL
```

Comment:
I don't get the point of using an UNION ALL here :man_shrugging:

Reply:
It is the header of the CSV file

Reply:
I agree, it's a bit of cheating but since it's only a migration script, I don't really mind.